### PR TITLE
Added PrioritisePD tag and relevant code required to get it working

### PR DESCRIPTION
**Pull request by seanra94**

### CODE REVIEW

**Comment by seanra94** [comment]
Ooops, made PR for wrong repo haha. Well, feel free to add my changes if you like :) I wanted to add the option to have PD weapons focus on their PD duties whilst still being available for targetting ships otherwise

File: src/main/kotlin/com/dp/advancedgunnerycontrol/typesandvalues/WeaponAITags.kt [hunk 1]
```diff
@@ -10,7 +10,7 @@ import com.fs.starfarer.api.Global
 import com.fs.starfarer.api.combat.WeaponAPI
 import com.fs.starfarer.api.fleet.FleetMemberAPI
 
-val pdTags = listOf("PD", "NoPD", "PD(Flx>N%)")
+val pdTags = listOf("PD", "NoPD", "PD(Flx>N%), PrioritisePD")
```
Comment (DesperatePeter) [inline src/main/kotlin/com/dp/advancedgunnerycontrol/typesandvalues/WeaponAITags.kt:13]: I feel like you are missing a pair of quotation marks here

**Comment by DesperatePeter** [comment]
Please also add the mode to the readme and list it as a potential option in the settings-portion of build.gradle.kts

File: src/main/kotlin/com/dp/advancedgunnerycontrol/typesandvalues/WeaponAITags.kt (hunk 1 above)
Comment (seanra94) [reply src/main/kotlin/com/dp/advancedgunnerycontrol/typesandvalues/WeaponAITags.kt:13]: Well that is embarrassing xD

**Comment by seanra94** [comment]
> Please also add the mode to the readme and list it as a potential option in the settings-portion of build.gradle.kts

Oh you're open to PRs? That's great! In that case, lemme decline this and ensure my PRs are up to scratch before posting them here :)

**Comment by DesperatePeter** [comment]
> > Please also add the mode to the readme and list it as a potential option in the settings-portion of build.gradle.kts
> 
> Oh you're open to PRs? That's great! In that case, lemme decline this and ensure my PRs are up to scratch before posting them here :)

Sure, why not =)
Alright, looking forward to it =)

**Pull request closed without merging**